### PR TITLE
Georgian Translation

### PR DESCRIPTION
**Pull request by Gr1ZzLy23**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added...
- Fixed...
- Added Georgian Translation

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by gave92** [comment]
Hello thanks for this contribution. Are you planning to translate the rest of the strings in this PR?

**Comment by Gr1ZzLy23** [comment]
Yes, I translated 500 lines and when I will finish it translation, I'll
send it ;)

File: Files.Package/Files.Package.wapproj [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <DisableXbfLineInfo>False</DisableXbfLineInfo>
     <GenerateTestArtifacts>True</GenerateTestArtifacts>
     <AppxBundleAutoResourcePackageQualifiers>Scale|DXFeatureLevel</AppxBundleAutoResourcePackageQualifiers>
-    <AppxDefaultResourceQualifiers>Language=en-US;ar;de-DE;es-ES;es-419;fr-FR;he-IL;hi-IN;hu-HU;id-ID;it-IT;ja-JP;ko-KR;lv-LV;nl-NL;or-IN;pl-PL;pt-BR;pt-PT;ru-RU;sv-SE;cs-CZ;ta;tr-TR;uk-UA;zh-Hans;zh-Hant;da;da-DK</AppxDefaultResourceQualifiers>
+    <AppxDefaultResourceQualifiers>Language=en-US;ar;de-DE;es-ES;es-419;fr-FR;he-IL;hi-IN;hu-HU;id-ID;it-IT;ja-JP;ka;ko-KR;lv-LV;nl-NL;or-IN;pl-PL;pt-BR;pt-PT;ru-RU;sv-SE;cs-CZ;ta;tr-TR;uk-UA;zh-Hans;zh-Hant;da;da-DK</AppxDefaultResourceQualifiers>
```
Comment (tsvietOK) [inline Files.Package/Files.Package.wapproj:58]: Same change should be done in https://github.com/files-community/Files/blob/main/Files/Files.csproj#L14

**Comment by Gr1ZzLy23** [comment]
What do you mean by that?

ხუთ, 29 ივლ. 2021, 00:19-ში Vladyslav Tsvietkov-მა ***@***.***>
დაწერა:

**Review by yaichenbaum — APPROVED** [review]
@Gr1ZzLy23 thank you!

**Pull request merged**